src/Dashboard/Search/KingRichardsKnight/Solution.java

```diff
@@ -0,0 +1,123 @@
+import java.io.*;
```

Comment:
Remove unused packages import, and try to avoid importing using all (*) as much as possible.